awscli/customizations/cloudformation/deployer.py

```diff
@@ -132,9 +132,10 @@ def wait_for_changeset(self, changeset_id, stack_name):
         # Wait for changeset to be created
         waiter = self._client.get_waiter("change_set_create_complete")
         # Poll every 5 seconds. Changeset creation should be fast
-        waiter.config.delay = 5
+        waiter_config = { 'Delay': 5 }
```

Comment:
This goes against PEP8. Make sure to remove the space before and after the closing and start brackets respectively.